src/isIsogram.test.js

```diff
@@ -8,12 +8,30 @@ describe(`Function 'isIsogram':`, () => {
   });
 
   it(`should return a boolean`, () => {
+    expect(typeof isIsogram('some string')).toBe('boolean');
+  });
+
+  it(`should return a 'true' if input ''`, () => {
+    expect(isIsogram('')).toBe(true);
+  });
 
+  it(`should accept letters in Latin`, () => {
+    expect(isIsogram('abc')).toBe(true);
   });
 
-  it(`should be case sensitive`, () => {
+  it(`should accept letters in Cyrillic`, () => {
+    expect(isIsogram('абв')).toBe(true);
+  });
 
+  it(`should be case sensitive when input in capital letters`, () => {
+    expect(isIsogram('Abc')).toBe(true);
   });
```

Comment:
Here you don't check if it's case sensitive because there are no same letters in different cases (like 'A'/'a'). You can remove this test